File: TransCoder_tokenized_test_set_functions/java/SHORTEST_COMMON_SUPERSEQUENCE.java
static int superSeq ( String X, String Y, int m, int n ) {
  if ( m == 0 ) return n;
  if ( n == 0 ) return m;
  if ( X . charAt ( m - 1 ) == Y . charAt ( n - 1 ) ) return 1 + superSeq ( X, Y, m - 1, n - 1 );
  return 1 + Math . min ( superSeq ( X, Y, m - 1, n ), superSeq ( X, Y, m, n - 1 ) );
}

File: TransCoder_tokenized_test_set_functions/cpp/SHORTEST_COMMON_SUPERSEQUENCE.cpp
int superSeq ( char * X, char * Y, int m, int n ) {
  if ( ! m ) return n;
  if ( ! n ) return m;
  if ( X [ m - 1 ] == Y [ n - 1 ] ) return 1 + superSeq ( X, Y, m - 1, n - 1 );
  return 1 + min ( superSeq ( X, Y, m - 1, n ), superSeq ( X, Y, m, n - 1 ) );
}